src/molecules/Tabs/Tabs.native.js

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useEffect } from 'react';
+import { Dimensions } from 'react-native';
+import PropTypes from 'prop-types';
+import { TabView } from 'react-native-tab-view';
+import { getIconNames } from '../../_helpers/icon';
+import isDefined from '../../_helpers/isDefined';
+import TabBar from './TabBar';
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+const Tab = () => {
+  return null;
+};
+
+Tab.propTypes = {
+  label: PropTypes.string.isRequired,
+  onPress: PropTypes.func,
+  disabled: PropTypes.bool,
+  icon: PropTypes.oneOf(getIconNames()),
+  testID: PropTypes.string,
+};
+
+Tab.defaultProps = {
+  onPress: () => {},
+  disabled: false,
+  testID: 'ds-tab-button',
+};
+
+const getTabs = ({ children }) => {
+  // Return only Tab Elements
+  return React.Children.toArray(children).filter(
+    (child) => child.type.name && child.type.name === 'Tab',
```

Comment:
```suggestion
    (child) => child && child.type === 'Tab',
```

Reply:
<img width="230" alt="Screenshot 2020-05-04 at 2 43 50 PM" src="https://user-images.githubusercontent.com/24487274/80951518-add38780-8e15-11ea-84ae-bdd8aaa5853e.png">

Reply:
Oh okay you're doing `.toArray` it's available on `children.map`